tests/util/zookeeper_test.py

```diff
@@ -23,6 +23,15 @@
 from kafka_utils.util.zookeeper import ZK
 
 
+def dump_to_json_and_encode(obj):
```

Comment:
Commented above. We could move this function in the kafka_utils/util/utils.py and implement it only once.